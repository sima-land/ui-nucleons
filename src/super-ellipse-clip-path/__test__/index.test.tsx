import { shallow } from 'enzyme';
import { SuperEllipseClipPath } from '..';

describe('SuperEllipseClipPath', () => {
  it('should match snapshot', () => {
    const component = shallow(<SuperEllipseClipPath id='test' />);
    expect(component.find('clipPath').prop('id')).toEqual('test');
    expect(component).toMatchSnapshot();
  });
});
