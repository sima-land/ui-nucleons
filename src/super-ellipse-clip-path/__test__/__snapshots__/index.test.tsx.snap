// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SuperEllipseClipPath should match snapshot 1`] = `
<div>
  <svg
    height="0"
    style="position: absolute;"
    width="0"
  >
    <defs>
      <clippath
        clipPathUnits="objectBoundingBox"
        id="test"
      >
        <path
          d="M1 0.5 C 1 0.856, 0.856 1, 0.5 1 S 0 0.856, 0 0.5 S 0.143 0, 0.5 0 S 1 0.143 1 0.5 Z"
        />
      </clippath>
    </defs>
  </svg>
</div>
`;
